rs/execution_environment/src/execution/install_code.rs

```diff
@@ -510,34 +478,6 @@ impl InstallCodeHelper {
 
         validate_controller(&self.canister, &original.sender)?;
 
-        validate_canister_settings(
```

Comment:
This is not needed as compute and memory allocations cannot be changed via `install_code` after this change. No point in checking settings then, as they cannot change anymore during `install_code`.